pom.xml

```diff
@@ -16,10 +17,11 @@
         <maven.compiler.target>11</maven.compiler.target>
         <postgresql.version>42.2.19</postgresql.version>
         <liquibase.version>4.3.2</liquibase.version>
+        <spring.data.jpa>2.4.4</spring.data.jpa>
+        <liquibase.version>4.3.2</liquibase.version>
```

Comment:
excessive property. it's already added above

Reply:
done